editor/plugins/gradient_editor.cpp

```diff
@@ -52,12 +52,12 @@ void GradientEditor::reverse_gradient() {
 
 int GradientEditor::_get_point_from_pos(int x) {
 	int result = -1;
-	int total_w = get_size().width - get_size().height - draw_spacing;
+	int total_w = get_size().width - get_size().height - draw_spacing - handle_width;
 	float min_distance = 1e20;
 	for (int i = 0; i < points.size(); i++) {
 		// Check if we clicked at point.
 		float distance = ABS(x - points[i].offset * total_w);
-		float min = (handle_width / 2 * 1.7); // Make it easier to grab.
+		float min = handle_width * 0.8; // Make it easier to grab.
```

Comment:
I would leave it in a `(handle_width / 2) * some_factor` form. With `some_factor > 1` it's easier to understand this scaling is actually an enlargement, what is enlarged, and by how much. That `* 0.8` got me thinking for a moment: _"wait, how is muliplying by less than one expanding the grab area?!"_.

Besides that `handle_width / 2` is an integer division so if the input/drawing/other code is using this then the same should be used in here, as the result can be different (by `0.5`) than for `handle_width / 2.0` or when simplified to something like `handle_width * 0.8`.

```suggestion
		float min = (handle_width / 2) * 1.6; // Make it easier to grab.
```

Reply:
Clarified this with a comment.